testgrid/cmd/gatherer/README.md

```diff
@@ -0,0 +1,18 @@
+#GitHub Issue Gatherer
+The GitHub Issue Gatherer collects issues and identifies issues that reference TestGrid tests. This is so that TestGrid can hyperlink failing or flaky tests to open issues.
+
+It has three parts: an Issue Scraper, a Target Identifier, and a Proto Writer.
+
+## Issue Scraper (issuescraper.go)
+The Issue Scraper uses the GitHub v3 API to collect every issue in a repository, and every comment left on those issues.
+
+GitHub calls Pull Requests "Issues", but this scraper doesn't.
+
+This scraper uses a caching layer, [ghCache](https://github.com/kubernetes/issues.json/tree/master/ghproxy/ghcache), to minimize repetitive API calls.
```

Comment:
Bad link.

Are you using the proxy cache as a run-once layer in the scraper? It would be better to have it live outside of and in front of  the scraper, persistently.

Reply:
The link is fixed, thanks for the catch

I intended for this object to stay alive with the object, and for GetIssues() to be called multiple times over its lifetime.